realm/realm-annotations-processor/src/main/java/io/realm/processor/Utils.kt

```diff
@@ -13,10 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- 
 package io.realm.processor
 
 import com.sun.tools.javac.code.Attribute
+import com.sun.tools.javac.code.Scope
```

Comment:
Not 100% sure I understand this. When I tried to use Java 11 to compile with...all of these imports failed. So not sure the `scopeElementsMethod` actually works with Java 9+?

Reply:
This PR doesn't update to build the SDK with JDK 11, but only fixes the cross module issue. This was just to keep the two issues separated as the JDK11 build support is maybe not a quick fix. 

The imports only fails when building the SDK with JDK11 but as the classes are in fact available in JDK11 the bytecode is almost OK. The reason why we need the differentiation is that the API has changed a bit, so some of the arguments have slightly different types. 

With the above fix I am able to build the example projects both with both JDK8 and JDK11. Without the reflective differentiation the build would work for JDK8 but would fail with 
```
Caused by: java.lang.NoSuchMethodError: com.sun.tools.javac.code.Symbol$TypeSymbol.members()Lcom/sun/tools/javac/code/Scope;
```
on JDK11.

Reply:
Building the annotation processor with JDK11 would render building the consumer projekt with JDK8 to fail with 
```
Caused by: java.lang.NoSuchMethodError: com.sun.tools.javac.code.Symbol$TypeSymbol.members()Lcom/sun/tools/javac/code/Scope$WriteableScope;
```
while the the JDK11 build would succeed.